docs/static/css/main.css

```diff
@@ -1772,6 +1839,508 @@
+      notes:
+        - "If the hero text is short or doesn't include a call to action, adding
+           the small-content class will increase the top padding value so the
+           text is more vertically aligned with the image."
+      codenotes:
+        - .hero__small-content
+  tags:
+    - cf-layout
+*/
+.hero {
+  background-color: #f1f2f2;
+}
+.hero_wrapper {
+  position: relative;
+  padding-top: 1.875em;
+  padding-bottom: 15.9375em;
+}
+.hero_wrapper:after {
+  display: block;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  background-position: center bottom 1.875em;
+  background-repeat: no-repeat;
+}
+@media only all and (min-width: 37.5em) {
+  .hero_wrapper {
+    padding-right: 20.8125em;
+    padding-bottom: 1.875em;
+  }
+  .hero_wrapper:after {
+    width: 333px;
+    background-position: center;
+  }
+}
+@media only all and (min-width: 50em) {
+  .hero_wrapper {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    min-height: 300px;
+    padding-top: 2.5em;
+    padding-right: 29.6875em;
+    padding-bottom: 2.5em;
+  }
+  .hero_wrapper:after {
+    width: 475px;
+    background-position: center;
+  }
+}
+.lt-ie9 .hero_wrapper {
+  min-height: 0;
+}
+.lt-ie9 .hero_wrapper:after {
+  background-position: right center;
+}
+.hero_heading {
+  font-family: "AvenirNextLTW01-Regular", Arial, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  margin-top: 0;
+  margin-bottom: 0.73076923em;
+  font-size: 1.625em;
+  line-height: 1.26923077;
+}
+.hero_heading em,
+.hero_heading i {
+  font-family: "AvenirNextLTW01-Italic", Arial, sans-serif;
+  font-style: italic;
+  font-weight: normal;
+}
+.lt-ie9 .hero_heading em,
+.lt-ie9 .hero_heading i {
+  font-style: normal !important;
+}
+.hero_heading strong,
+.hero_heading b {
+  font-family: "AvenirNextLTW01-Demi", Arial, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+.lt-ie9 .hero_heading strong,
+.lt-ie9 .hero_heading b {
+  font-weight: normal !important;
+}
+@media only all and (max-width: 37.4375em) {
+  .hero_heading {
+    font-family: "AvenirNextLTW01-Regular", Arial, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    margin-top: 0;
+    margin-bottom: 0.95454545em;
+    font-size: 1.375em;
+    line-height: 1.27272727;
+  }
+  .hero_heading em,
+  .hero_heading i {
+    font-family: "AvenirNextLTW01-Italic", Arial, sans-serif;
+    font-style: italic;
+    font-weight: normal;
+  }
+  .lt-ie9 .hero_heading em,
+  .lt-ie9 .hero_heading i {
+    font-style: normal !important;
+  }
+  .hero_heading strong,
+  .hero_heading b {
+    font-family: "AvenirNextLTW01-Demi", Arial, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+  }
+  .lt-ie9 .hero_heading strong,
+  .lt-ie9 .hero_heading b {
+    font-weight: normal !important;
+  }
+}
+@media only all and (max-width: 37.4375em) and only all and (max-width: 37.4375em) {
```

Comment:
Is this supposed to be a min and max query? And if so, the values should be different